SpatialGDK/Source/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -71,7 +71,7 @@ void USpatialPackageMapClient::RemoveEntityActor(Worker_EntityId EntityId)
 	}
 }
 
-FNetworkGUID USpatialPackageMapClient::ResolveStablyNamedObject(const UObject* Object)
+FNetworkGUID USpatialPackageMapClient::ResolveStablyNamedObject(UObject* Object)
```

Comment:
How was this enforced by 4.20?

Reply:
GetNetworkChecksum is now being used internally, and it doesn't take a const UObject.